Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -94,11 +94,27 @@ def putMaskOnVTKGrid(data, grid, actorColor=None, cellData=True, deep=True):
             grid.GetPointData().AddArray(ghost)
     return mapper
 
+def handleProjectionEdgeCases(projName, data):
+    # For mercator projection, latitude values of -90 or 90
+    # transformation result in infinity values. We chose -85, 85
```

Comment:
Technically, 85.051129 is used because that makes the domain a square in projected coordinates.
https://en.wikipedia.org/wiki/Web_Mercator#Formulas